### PR TITLE
Allow arbitrary output sizes

### PR DESCRIPTION
**Pull request by DJMcNab**
This just took some simple wiring through to pass the dynamic size down into `render_full`, and then blit the image into a new buffer of the output size.

The command can now be ran as `cargo run -p piet-wgsl -- [width] [height]`, which each default to `1024`.

### CODE REVIEW

File: piet-wgsl/src/render.rs [hunk 1]
```diff
@@ -52,7 +53,14 @@ fn size_to_words(byte_size: usize) -> u32 {
     (byte_size / std::mem::size_of::<u32>()) as u32
 }
 
-pub fn render(scene: &Scene, shaders: &Shaders) -> (Recording, BufProxy) {
+pub const fn next_multiple_of(val: u32, rhs: u32) -> u32 {
+    match val % rhs {
```
Comment (raphlinus) [inline piet-wgsl/src/render.rs:57]: One other way to write this is `val + rhs - 1 - (val + rhs - 1) % rhs`

**Review by raphlinus — APPROVED** [review]
Thanks! This is definitely a useful step toward making it dynamically sized for window-integration.

File: piet-wgsl/src/render.rs (hunk 1 above)
Comment (DJMcNab) [reply piet-wgsl/src/render.rs:57]: I copied this from the unstable `std` implementation, and this is not hot

I'd be inclined to leave it as it.

**Comment by DJMcNab** [comment]
Stupid. I targeted main again

**Comment by DJMcNab** [comment]
Fixed

**Pull request merged**